lib/AST/ASTPrinter.cpp

```diff
@@ -2963,34 +2963,10 @@ static bool usesFeatureStaticAssert(Decl *decl) {
 }
 
 static bool usesFeatureEffectfulProp(Decl *decl) {
-  if (auto asd = dyn_cast<AbstractStorageDecl>(decl))
-    return asd->getEffectfulGetAccessor() != nullptr;
   return false;
 }
 
 static bool usesFeatureAsyncAwait(Decl *decl) {
-  if (auto func = dyn_cast<AbstractFunctionDecl>(decl)) {
```

Comment:
It could be nice to have a brief comment in these `usesFeature` functions that explains that we're returning `false` because the features are old.

Reply:
I was thinking of adding a macro locally that documented this better, e.g.,

```swift
ASSUME_FEATURE_IN_ALL_SWIFT_COMPILERS(AsyncAwait)
```

Reply:
Yeah that seems like good option too.

Reply:
I went with the `BASELINE_LANGUAGE_FEATURE` approach in `Features.def`